diff -up kleopatra-20.03.90/src/conf/cryptooperationsconfigwidget.cpp.omv~ kleopatra-20.03.90/src/conf/cryptooperationsconfigwidget.cpp
--- kleopatra-20.03.90/src/conf/cryptooperationsconfigwidget.cpp.omv~	2020-04-13 01:07:20.780854120 +0200
+++ kleopatra-20.03.90/src/conf/cryptooperationsconfigwidget.cpp	2020-04-13 01:07:31.658974722 +0200
@@ -295,17 +295,9 @@ void CryptoOperationsConfigWidget::setup
 
     connect(mQuickSignCB, &QCheckBox::toggled, this, &CryptoOperationsConfigWidget::changed);
     connect(mQuickEncryptCB, &QCheckBox::toggled, this, &CryptoOperationsConfigWidget::changed);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(mChecksumDefinitionCB, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
-#else
-    connect(mChecksumDefinitionCB, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) ,
-#endif
             this, &CryptoOperationsConfigWidget::changed);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(mArchiveDefinitionCB, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
-#else
-    connect(mArchiveDefinitionCB, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) ,
-#endif
             this, &CryptoOperationsConfigWidget::changed);
     connect(mPGPFileExtCB, &QCheckBox::toggled, this, &CryptoOperationsConfigWidget::changed);
     connect(mAutoDecryptVerifyCB, &QCheckBox::toggled, this, &CryptoOperationsConfigWidget::changed);
