diff -up kleopatra-20.03.80/src/utils/dragqueen.cpp.omv~ kleopatra-20.03.80/src/utils/dragqueen.cpp
--- kleopatra-20.03.80/src/utils/dragqueen.cpp.omv~	2020-03-23 17:51:59.956485696 +0100
+++ kleopatra-20.03.80/src/utils/dragqueen.cpp	2020-03-23 17:52:08.628421289 +0100
@@ -208,7 +208,7 @@ void DragQueen::mouseMoveEvent(QMouseEve
             (e->buttons() & Qt::LeftButton) &&
             (m_dragStartPosition - e->pos()).manhattanLength() > QApplication::startDragDistance()) {
         QDrag *drag = new QDrag(this);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
         if (const QPixmap *const pix = pixmap()) {
             drag->setPixmap(*pix);
             drag->setHotSpot(calculate_hot_spot(e->pos(), pix->size(), this));
